.env-example

```diff
@@ -1 +0,0 @@
-UNSPLASH_API_KEY=xxx
```

Comment:
This line is necessary because the Unsplash API key is needed to load images from Unsplash service. The example file is meant to be copied into `.env` file, in order to use it locally. Please restore this line of code